src/main/scala/com/schuwalow/todo/repository/DoobieTodoRepository.scala

```diff
@@ -26,7 +27,7 @@ final class DoobieTodoRepository(xa: Transactor[Task]) {
 
   val todoRepository = new TodoRepository.Service {
 
-    def getAll(): UIO[List[TodoItem]] =
+    def getAll: UIO[List[TodoItem]] =
```

Comment:
nice cleanup.
these could be val as well

Reply:
I've just changed them to vals